lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -255,7 +304,19 @@ defmodule Membrane.MP4.Muxer.CMAF do
       |> then(&(Enum.sum(&1) / length(&1)))
       |> floor()
 
-    buffer = %Buffer{payload: payload, metadata: %{duration: duration}}
+    # there can be a lot of different tracks inside the segment, but if any of them starts with a key frame
+    # it means that we have an independent segment
+    independent =
+      Enum.all?(acc, fn {_pad, samples} ->
```

Comment:
Either it should be `Enum.any?` or the comment is wrong

Reply:
Yup, the comment is quite old compared with what the code ended to be, will change that.